restdocs-api-spec-jsonschema/src/test/kotlin/com/epages/restdocs/apispec/jsonschema/JsonSchemaFromFieldDescriptorsGeneratorTest.kt

```diff
@@ -168,6 +176,18 @@ class JsonSchemaFromFieldDescriptorsGeneratorTest {
         then(objSchema.requiredProperties).contains("obj")
     }
 
+    @Test
+    fun should_generate_schema_for_required_object_in_array() {
```

Comment:
`should_generate_schema_for_required_object_in_array`
->
`should_generate_schema_for_required_array_in_object`

The array is a required field in the top-level object.